torch/csrc/distributed/rpc/agent_utils.cpp

```diff
@@ -95,6 +96,10 @@ std::unordered_map<std::string, worker_id_t> collectCurrentNames(
     // workerInfos are comma separated, (e.g.
     // "Name1-Rank1,Name2-Rank2,Name3-Rank2") parse list of workers
     for (const std::string& workerInfo : splitString(allWorkerInfos, ",")) {
+      // Skip if the token is empty
+      if (workerInfo.empty()) {
```

Comment:
I assume this means the entire token is empty? If thats the case, the if block should be before the for loop?

Reply:
This string was formatted in such a way that the first token of the vector of tokens was empty (due to the initial comma). This was unclear so I changed `splitString()` to `splitStringRemoveEmpties()` to also remove empty strings.